verify/terraform/skus/current/windows-11-23H2/Makefile

```diff
@@ -9,7 +9,7 @@ SHELL=/bin/bash
 all: apply
 
 terraform.tfvars:
-	cat terraform.tfvars.template | sed -e "s/%password%/$$(pwgen -s --remove-chars="\'\"$$%{}" -y 20 1)/gi" > terraform.tfvars
+	cat ../../../modules/terraform.tfvars.template | sed -e "s/%password%/$$(pwgen -s --remove-chars=\'\"$$%{} -y 20 1)/gi" > terraform.tfvars
```

Comment:
It greatly appreciated trying to fix the issue, but there may be more simple solution.

(IMHO sed -e in Makefile cause a troublesome in careless)

How about just replace it?

```
	cat ../../../modules/terraform.tfvars.template | \
		sed `-e` "/^windows-password = \"%PASSWORD%\"/c windows-password = \"$$(pwgen -s --remove-chars=\" --symbols 20 1)\""> terraform.tfvars
```

Reply:
Thank you for your helpful suggestion!
I will take it--to be honest, I have less knowledge around the topic.

After confirming it works fine, I'd put another commit.

Reply:
I took the suggested fix and ran `make terraform.tfvars` command but ended up with a strange message saying like:

```
$ make terraform.tfvars
cat ../../../modules/terraform.tfvars.template | \
        sed `-e` "/^windows-password = \"%PASSWORD%\"/c windows-password = \"$(pwgen -s --remove-chars=\" --symbols 20 1)\""> terraform.tfvars
/bin/bash: line 2: -e: command not found
$
```

Confusing a bit;
I have no idea on what `-e: command not found` means.

To my eyes, `terraform.tfvars` itself was created successfully with an apparently expected windows-password string. 
So seemingly `make terraform.tfvars` worked fine.

Some websites from my search mention $PATH variables but I'm not sure if it is the reason why...

Reply:
sorry, I've wrongly pasted snippet.

Reply:
It should be something like:

```
cat ../../../modules/terraform.tfvars.template | \
              sed -e "/^windows-password = \"%PASSWORD%\"/c windows-password = \"$$(pwgen -s --remove-chars=\" -y 20 1)\"" > terraform.tfvars
```

Reply:
Superb!
It worked with no error message.
I've pushed another commit to include it.